### DISCUSSION TITLE
DSM 7 - will it work?

### DISCUSSION THREAD

**Discussion by Debug1Bug** [runfalk/synology-wireguard#77] [Q&A] [answered]
I decided to upgrade to DSM 7, hoping that wireguard might be included. Unfortunately it was not.
Do you know if it's safe to put this kernel extension on DSM 7? (Synology DS1819+)

**Comment by runfalk** [comment, marked as answer]
It doesn't work OOTB, but @Matige has made a PR (https://github.com/runfalk/synology-wireguard/pull/71) that seems to work. You'll need to checkout their branch and build the package yourself. There should be instructions in the README.

I haven't merged that PR yet because I haven't been able to test it.

**Comment by Debug1Bug** [comment]
Ok, thanks. I will try to figure it out.

**Comment by LucasJanin** [comment]
Any update on the DSM 7 support? Thanks

**Reply by meptalon** [reply]
+1 would love to have a signed DSM 7.0 package. I didn't manage to compile my own with the instructions around (DS220+ for me)
